api/client/volume/create.go

```diff
@@ -60,3 +61,34 @@ func runCreate(dockerCli *client.DockerCli, opts createOptions) error {
 	fmt.Fprintf(dockerCli.Out(), "%s\n", vol.Name)
 	return nil
 }
+
+var createDescription = `
+Creates a new volume that containers can consume and store data in. If a name is not specified, Docker generates a random name. You create a volume and then configure the container to use it, for example:
```

Comment:
We stick to 80 chars, perhaps this slipped through during review, but let's not inherit that mistake